src/slatracker/urls.py

```
@@ -20,4 +20,7 @@
 
 urlpatterns = [
     path("admin/", admin.site.urls),
+    path('api/v1/ticket-events/', TicketEventIngestView.as_view(), name='ticket-ingest'),
```

Ruff (0.11.9)

Undefined name `TicketEventIngestView`

(F821)